src/eterna/pose2D/HighlightBox.ts

```diff
@@ -14,19 +22,29 @@ export enum HighlightType {
     UNSTABLE = 5,
     SHIFT = 6,
     USER_DEFINED = 7,
+    ANNOTATION = 8,
 }
 
 /** A class for highlighting groups of bases in a Pose2D */
-export default class HighlightBox extends GameObject implements LateUpdatable {
```

Comment:
If I remember correctly, it did not seem as though GameObject instances received interaction events (pointerOver, pointerOut) that were central to some HighlightBox behavior I wanted to include. That being said, I could have inherited the original HighlightBox (used by some other existing infrastructure) instead of augmenting it as I had done here. Not sure if we have a preference

Reply:
What about making this a GraphicsObject? It's managing a Graphics anyways, and then you can avoid the intermediate container with only one child

Reply:
Good suggestion. Seems to work just fine